pychromecast/socket_client.py

```diff
@@ -281,7 +301,7 @@ def send_message(self, destination_id, namespace, data,
         msg.destination_id = destination_id
         msg.payload_type = cast_channel_pb2.CastMessage.STRING
         msg.namespace = namespace
-        msg.payload_utf8 = json.dumps(data, ensure_ascii=False).encode("utf8")
+        msg.payload_utf8 = json.dumps(data)
```

Comment:
I am not sure if it expects or Strings. Let's go with your proposed solution for now. We'll have to do a full regression anyway when Protobuf 3 gets released.